index.bs

```diff
@@ -4580,6 +4681,13 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
 1. If [=user verification=] is required for this registration, verify that the [=User Verified=] bit of the <code>[=flags=]</code>
     in |authData| is set.
 
+1. If the credential [=backup eligibility=] is used as part of Relying Party business logic or policy, evaluate the 
+    [=backup eligibility=] bit of the <code>[=flags=]</code> in |authData|.
```

Comment:
In addition to @emlun's comments, I note we recommend (way back up on line 3728) the RP storing both backup flags/bits, but here in the RP ops sections we are recommending storing only the "backup state" and not "backup eligibility".  An RP may wish to store both values, e.g., to inform their UX flows?  (use of various forms of the term "... flow(s)" in various places in the spec is established)

Additionally, by "business logic" do you mean UX flows?  We have not used the term "business logic" in the spec as yet. these cred backup-driven additions to the RP ops are the only places to do so.  

Perhaps:
```suggestion
1. If the [=[RP]=] uses the credential's [=backup eligibility=] to inform its user experience flows and/or policies, evaluate the 
    [=backup eligibility=] bit of the <code>[=flags=]</code> in |authData|.
```

Reply:
Business logic is the traditional way to refer to this, but I have no issue with your proposed text.